services/web/server/tests/unit/with_dbs/03/login/test_login_2fa.py

```diff
@@ -242,6 +245,55 @@ def _get_confirmation_link_from_email():
     assert user["phone"] == fake_user_phone_number
     assert user["status"] == UserStatus.ACTIVE.value
 
+    # login (via EMAIL) ---------------------------------------------------------
```

Comment:
Take the opportunity to split this test, please. Try to set a different test for each use case.